src/Illuminate/Support/Number.php

```diff
@@ -24,6 +24,17 @@ class Number
      */
     protected static $currency = 'USD';
 
+    /**
+     * Get a new numberable object from the given number.
+     *
+     * @param  int|float|string  $number
```

Comment:
You can narrow this further down:

```suggestion
     * @param  int|float|numeric-string  $number
```